### PR TITLE
{lang}[foss/2018a] R 3.4.4

### PR DESCRIPTION
**Pull request by RvDijk**
libgeotiff depedency was added to the GDAL module, since the R package `rgdal` was linking to `/usr/lib64/libgeotiff.so`, which resulted in errors.

### CODE REVIEW

**Comment by RvDijk** [comment]
@boegel ping?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
node2452.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b84231614ca0d9e8d520e5cdc34792f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a5983a6c4559192fd11567bcfc977d50 for a full test report.

**Comment by boegel** [comment]
@RvDijk Sorry for not responding to this earlier... Test reports are coming up, if those don't raise any red flags this is good to go (to be included in EasyBuild v3.6.1).

File: easybuild/easyconfigs/r/R/R-3.4.4-foss-2018a-X11-20180131.eb [hunk 1]
```diff
@@ -0,0 +1,1811 @@
...
+        'checksums': ['c0a71090d5b70b9a95a6936091dabae9c26e1fc6b9609bfe5fb6346033905e48'],
+    }),
+    ('doMC', '1.3.5', {
+        'checksums': ['792254596babd8cc074bb3c83c18672dd07a0c3246b4ebab08d8ebedf6f4d9ed'],
+    }),
+    ('DBI', '0.8', {
+        'checksums': ['d52b9e04b3291f9bda8a88c174678f520ddffc5f2edf9b3dfa6d97dca943ce9a'],
+    }),
+    ('foreign', '0.8-69', {
+        'checksums': ['820e1409162fbd6418b80a0f5a4b50d649ee74f06d195a5210745619b16e1592'],
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.4.4-foss-2018a-X11-20180131.eb]: This should be `13689f5ec1ab09e8973be81c7f1799b7de4313176072887a9fa0b5825aed3468`, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6245

File: easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.4.2-foss-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+name = 'libgeotiff'
+version = '1.4.2'
+
+homepage = 'https://directory.fsf.org/wiki/Libgeotiff'
+description = "Library for reading and writing coordinate system information from/to GeoTIFF files"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [
+    'http://download.osgeo.org/geotiff/libgeotiff/'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.4.2-foss-2018a.eb]: @RvDijk Missing `,` after this line (which makes the download fail)

File: easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.4.2-foss-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    ('LibTIFF', '4.0.9'),
+    ('PROJ', '5.0.0'),
+    ('libjpeg-turbo', '1.5.3'),
+]
+
+configopts = ' --with-libtiff=$EBROOTLIBTIFF --with-proj=$EBROOTPROJ --with-zlib=$EBROOTZLIB'
+configopts += ' --with-jpeg=$EBROOTLIBJPEGMINTURBO'
+
+sanity_check_paths = {
+    'files': ['bin/listgeo'],
```
Comment (boegel) [inline easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.4.2-foss-2018a.eb]: @RvDijk Please enhance to also check libraries and non-empty `include` dir:

```python
sanity_check_paths = {
    'files': ['bin/listgeo', 'lib/libgeotiff.a', 'lib/libgeotiff.%s' % SHLIB_EXT],
    'dirs': ['include', 'share'],
}
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2409.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/34d5002c9b711017b990762ab9daafbf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2114.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6110eca5737a6d9a41fca051c8369a43 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**